### DISCUSSION TITLE
I have a question

### DISCUSSION THREAD

**Discussion by CYWWEZ** [EvanMulawski/BeamMP.EnvSync#12] [General]
How to install this plugin to beammp

**Comment by EvanMulawski** [comment]
Hi, please refer to the [Installation](https://github.com/EvanMulawski/BeamMP-EnvSync/wiki/Installation) page on the wiki.

**Reply by CYWWEZ** [reply]
Is it copied directly to the server directory

**Comment by CYWWEZ** [comment]
My will prompt Error Backend Response Failed to PARSE As Valid JSON
